scalactic.dotty/src/main/scala/org/scalactic/anyvals/RegexString.scala

```diff
@@ -125,10 +125,10 @@ private[scalactic] final class RegexString private (val value: String) extends A
     value.replaceFirst(regex, replacement)
 
   def split(regex: String): Array[String] =
-    value.split(regex)
+    value.split(regex).map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.